.github/workflows/update-v8.yml

```
@@ -0,0 +1,66 @@
+name: V8 update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
+
+  workflow_dispatch:
+    inputs:
+      id:
+        description: The ID of the job to run
+        required: true
+        default: all
+        type: choice
+        options:
+          - all
+          - minor
+          # preparing for v8 major
+permissions:
+  contents: read
+
+jobs:
+  v8-update:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false  # Prevent other jobs from aborting if one fails
+      matrix:
+        include:
+          - id: minor
```

```suggestion
          - id: patch
```

I made a mistake by calling it minor in ncu.